frontend/src/features/projects/dispose/constants.tsx

```diff
@@ -0,0 +1,12 @@
+import { DisposeWorkflowStatus, ProjectWorkflowComponent } from '.';
+import React from 'react';
+
+//TODO: map to real components
+export const projectWorkflowComponents: ProjectWorkflowComponent[] = [
```

Comment:
Is this the way you are mapping components to project status?  Instead of the `route` value in the db?